shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -281,6 +284,43 @@ public boolean isFailure() {
         }
     }
 
+    @Data
+    public static final class NotEmptyTableSegment implements Reply, WireCommand {
```

Comment:
I think this should be called "TableSegmentNotEmpty"

Reply:
done.